CoreArrangementPlugins/src/au/gov/asd/tac/constellation/plugins/arrangements/ArrangementPluginRegistry.java

```diff
@@ -70,6 +71,7 @@
     public static final String MDS = ArrangeInMDSPlugin.class.getName();
     public static final String PENDANTS = PendantsArranger.class.getName();
     public static final String RANDOM = RandomArrangementPlugin.class.getName();
+    public static final String SCATTER3D = ArrangeInScatter3dGeneralPlugin.class.getName();
```

Comment:
My brain really wants to think this says scattered

Reply:
Can you change it to `SCATTER_3D` ?